ForkClone.md

```diff
@@ -1,33 +1,33 @@
-# Fork and Clone 
+# Fork and clone 
 
-## Steps to fork the repo: 
+## Steps to fork the repository (repo): 
 
 1. Go to the main GitHub page of the repo:
     (https://github.com/gbowne1/codebooker/)
    
-1. Once you are on the main GitHub page of the repo, click the **Fork** button at the top right of the page. 
+1. Select **Fork** at the top right of the page. 
 
     ![](fork.png)
 
-1. You will now be on a page with the heading **Create a new fork**.
+    This opens a page with the heading **Create a new fork**.
 
-     There are two fields which are required, Owner and Repository name. Make sure the owner is you and the repository name is correct. These fields should already be populated when you clicked to this page in step 2.
+1. Review the prepopulated fields to make sure the owner is you and the repo name is correct.
```

Comment:
There are many ways to say this. The reason I changed it the way I did was to emphasize the action the user needs to take. Other benefits are that it removes "should," which many style guides say to avoid, and it meant I didn't have to worry about punctuating the original first sentence. 

(For context, "which are required" is a nonrestrictive clause, which means it requires a comma before it. If there are only two fields and both are required, that's how we must punctuate it. If there are multiple fields, but only two are required, we'd change "which" to "that" and leave it without a comma.)